frontend/src/metabase/api/activity.ts

```diff
@@ -1,22 +1,35 @@
 import type {
   RecentItem,
+  CreateRecentRequest,
+  RecentsResponse,
   PopularItem,
-  RecentItemsResponse,
   PopularItemsResponse,
+  RecentsRequest,
 } from "metabase-types/api";
 
 import { Api } from "./api";
-import { provideActivityItemListTags } from "./tags";
+import {
+  provideActivityItemListTags,
+  invalidateTags,
+  idTag,
+  TAG_TYPE_MAPPING,
+} from "./tags";
 
 export const activityApi = Api.injectEndpoints({
   endpoints: builder => ({
-    listRecentItems: builder.query<RecentItem[], void>({
-      query: () => ({
-        method: "GET",
-        url: "/api/activity/recent_views",
-      }),
-      transformResponse: (response: RecentItemsResponse) =>
-        response?.recent_views,
+    listRecents: builder.query<RecentItem[], RecentsRequest | void>({
+      query: ({ context } = { context: ["views"] }) => {
+        const contextParam = [...context]
+          .sort()
+          .map(ctx => `context=${ctx}`)
+          .join("&");
```

Comment:
nit, nonblocking, using `URLSearchParams` might be tidier and more explicit here

Reply:
`api` supports params and just body; both will handle query string params for `GET`